python/paddle/fluid/tests/unittests/white_list/check_shape_white_list.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+NOT_CHECK_OP_LIST = [
+    # ngraph: increment's input must be 1-d and only has one data
+    'increment'
```

Comment:
@baojun-nervana @LeoZhao-Intel Please review it

Reply:
Does ngraph increase op cause any problem here? You may disable increase op by comment out this line https://github.com/NervanaSystems/ngraph-paddle/blob/develop/paddle/fluid/operators/ngraph/ops/increment_op.h#L49. We then can submit following PR to fix the op.

Reply:
can author provide the use case or expected behavior for increase op? not very clear if just judge from comment here:

# ngraph: increment's input must be 1-d and only has one data

and even from this PR, I didn't find NOT_CHECK_OP_LIST is used at all.

Reply:
@LeoZhao-Intel We want each Op to contain a unit test with dimensions of input >=100.
@baojun-nervana You can remove `increment` here, and you may find the problem.

Reply:
how to run this  unittest? can yo share cmd?

Reply:
`ctest -R test_increment_ngraph_op -V`

Reply:
seems increment op has restriction on input dim, must be 1.

https://github.com/NervanaSystems/ngraph-paddle/blob/develop/paddle/fluid/operators/increment_op.cc#L34

so test case may be invalid.

Reply:
https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api_cn/layers_cn/increment_cn.html

Reply:
Got it. Thanks @LeoZhao-Intel 
> ngraph: increment's input must be 1-d and only has one data

@zhupengyang the notes is not correct, it is not just for ngraph. see https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api_cn/layers_cn/increment_cn.html